.editorconfig

```diff
@@ -7,8 +7,9 @@ end_of_line = lf
 trim_trailing_whitespace = true
 insert_final_newline = true
 
-[*.ts]
+[*.ts,*.java]
```

Comment:
Trying to fix java files to use spaces and not tabs.  However this still doesn't fix it.  Not certain why.  Need to fix these issues before turning on linting rules in CI/CD.